src/com/twitter/intellij/pants/execution/PantsMakeBeforeRun.java

```diff
@@ -199,6 +199,7 @@ public boolean executeTask(
     // If project has not changed since last Compile, return immediately.
     if (!FileChangeTracker.shouldRecompileThenReset(currentProject, targetAddressesToCompile)) {
       PantsExternalMetricsListenerManager.getInstance().logIsPantsNoopCompile(true);
+      showPantsMakeTaskMessage("Already up to date.\n", ConsoleViewContentType.SYSTEM_OUTPUT, currentProject);
```

Comment:
It is related, because if user triggers the same action without any change, the IDE will noop, which makes a bit confusing.